config/routes.rb

```diff
@@ -1,3 +1,33 @@
+<<<<<<< HEAD
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  devise_for :users, controllers: { confirmations: "user/confirmations",
+    passwords: "user/passwords",
+    registrations: "user/registrations",
+    sessions: "user/sessions",
+    unlocks: "user/unlocks",
+    omniauth_callbacks: "user/omniauth_callbacks",
+  }
+
+  resources :users, only: [:show]
+  resources :posts, only: [:new, :create, :destroy, :edit, :update]
+
+  devise_scope :user do
+    authenticated :user do
+      root 'feed#index', as: :authenticated_root
+    end
+
+    unauthenticated do
+      root 'user/sessions#new', as: :unauthenticated_root
+    end
+  end
+  get "/feed", to: "feed#index"
+=======
+# frozen_string_literal: true
+
+#  Routes for our fakebook app
+Rails.application.routes.draw do
+  root to: 'home#signin'
```

Comment:
I get an error when I try to check the root url in the browser, see #5

Reply:
when fixing the conflict this is changed